tasks/pip.yml

```
@@ -328,3 +328,18 @@
   register: result
   until: result is success
   notify: reload supervisor
+
+- name: Install django
+  pip:
+    name: "django~=3.2.0"
```

@yashikajotwani12  we need to add a variable that contains `~=3.2.0`. 

```suggestion
    name: "django{{ openwisp2_django_version}}"
```

This variable should be added in https://github.com/openwisp/ansible-openwisp2/blob/master/defaults/main.yml with the default value of `~=3.2.0`. 

Please also update the ["Role Variables" section of the README](https://github.com/openwisp/ansible-openwisp2#role-variables) to add this new variable.

---

Yeah! I will update these changes soon, Thanks